src/watchmaker/utils/urllib/request_handlers.py

```diff
@@ -8,20 +8,13 @@
 
 from six.moves import urllib
 
-HAS_BOTO3 = False
-try:
-    import boto3
```

Comment:
I _think_ this module might now be able to import boto3 unconditionally, because this module is only ever referenced by `watchmaker.utils.urllib` and that is already gated.

Reply:
I removed the check and import, added the import back.

Reply:
Actually, we do need the try surrounding the import, not the HAS_BOTO3 line as that was removed to another spot, but without the try except it fails all unit tests

Reply:
Hmm, can you check again exactly what was the error? While this works, I feel like maybe there should be a different fix, now that the condition is being handled elsewhere...

Reply:
`tests/test_utils.py:7: in <module>
    import watchmaker.utils
.tox/3.6-codecov/lib/python3.8/site-packages/watchmaker/__init__.py:18: in <module>
    import watchmaker.utils
.tox/3.6-codecov/lib/python3.8/site-packages/watchmaker/utils/__init__.py:13: in <module>
    from watchmaker.utils import urllib
.tox/3.6-codecov/lib/python3.8/site-packages/watchmaker/utils/urllib/__init__.py:10: in <module>
    from watchmaker.utils.urllib.request_handlers import S3Handler
.tox/3.6-codecov/lib/python3.8/site-packages/watchmaker/utils/urllib/request_handlers.py:10: in <module>
    import boto3  # type: ignore
E   ModuleNotFoundError: No module named 'boto3'`

The test discovery fails in tox as well as in vscode.

Reply:
hmm, this stands out to me:

```
from watchmaker.utils.urllib.request_handlers import S3Handler
```

So I think then in `watchmaker.utils.urllib`, we can move the import into the gated check...

```
if HAS_BOTO3:
    from watchmaker.utils.urllib.request_handlers import S3Handler
    request.install_opener(request.build_opener(S3Handler))
```

Reply:
The above change was made, but the goal of that was to allow this module to `import boto3`, without gating it on the try/except.